### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by joshmccall221** [CDSGlobal/CDS-Payments-Widget#1]
Payment Widget – Minimum Requirements for Initial Client Onboarding

ToDo:
```
Out of Scope (Dependent on Phase 2 of Tokenization being released)
- CVV
- AVS
Will not need the "Name" Field to be displayed
Real-time Authorizations
```

Done / Need reviewed:
```
Technical Requirements / Where the Widget will be stored/served from?
[x] i.e - AWS/S3 Bucket or somewhere else
[x] Will CloudFront be used to provide meaningful URL Address?
http://d2znaubcly5dbd.cloudfront.net/index.html
Onboarding Information / Integration Guide
[x] Needs to include Browser Encryption guide for a standalone integration
[x] Needs to instruct client on how to put the Payment Widget on their acquisition page
Default Configuration 
[x] Will not be able to be modified by client
For the initial offering, the following configurations
[x] VISA, MasterCard, American Express, Discover
[x] VISA, MasterCard, American Express
[x] VISA, MasterCard, Discover
[x] VISA, MasterCard, American Express, Discover, Bill-me
[x] VISA, MasterCard, American Express, Bill-me
[x] VISA, MasterCard, Discover, Bill-me
User Interface / Spacing
[x] Reduce White Space
[x] Allow up to 4 Card Brands on the line in the Card Animation before breaking lines
*NOTE*: with responsive sizing this is next to impossible to enforce
[x] Take away in focus/highlighting of the card icons when mouse is hovered over icon or when icon is clicked on – it appears to have no effect
[-] Do not wrap credit card number on Card Animation
*NOTE* This is a library and largely out of our control. We can discuss more during update meeting
Grouping
[x] Eliminate top level/outer level container so we avoid the "Box within a box" effect
User Feedback
Card Animation
[x] Show Card Brand once BIN entered
[x] Show Expire Date when date entered
Messaging/Validation
[x] Type/Valid – remove these fields on the finished product.
[x] Expire Date – need validation 
[x] Do not accept a past date and/or too far in the future
*NOTE* past dates are not accepted and defaults to 19 years in the future
[x] Do not accept an invalid month – i.e. 18
[x] Make sure that 4 digit and 2 digit years are being handled correctly – it looks like it can be entered either way
[x] When Credit Card information is typed in, disable Bill Me Option and vice versa when Bill Me Option is clicked.
```

Baring feedback the items left is documentation and deploying / testing.

Demo link: 
[demo.cds-payments.surge.sh](http://demo.cds-payments.surge.sh/) 

[cdsglobal.github.io/CDS-Payments-Widget/](https://cdsglobal.github.io/CDS-Payments-Widget/)

**Comment by joshmccall221** [comment]
Split into individual issues... closing
